bcrs-ui/src/router/index.ts

```diff
@@ -0,0 +1,13 @@
+import Vue from 'vue'
+import VueRouter from 'vue-router'
+import { routes } from './routes'
+
+Vue.use(VueRouter)
+
+const router = new VueRouter({
+  mode: 'history',
+  base: process.env.BASE_URL,
+  routes
+})
```

Comment:
I think you should add the "scrollBehaviour" attribute here (same as coops-ui). Thoughts?

Reply:
Yes i do believe that is a good idea. Interesting that it isn't the default behaviour, maybe i will do a little research.